packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -78,6 +93,25 @@ export default function ButtonAppBar({
           </Grid>
         </Grid>
       </Toolbar>
+      <div>
+        <SideDrawer
+          isOpen={showSideBar}
+          close={() => setShowSideBar(false)}
+          navProps={{
+            navItemsProps: [
```

Comment:
Please add the about page (here or in another PR)

Reply:
done